test/integration/config.test.ts

```diff
@@ -154,31 +153,6 @@ test('it loads .ts files that import from internal modules', async () => {
   )
 })
 
-test('it loads json files', async () => {
```

Comment:
When I removed the line with condition for json format from `loadConfig` function, this test was still passing and the json content was bundled into JS file. So, I'm wondering if we need to tweak loading and resolving logic a bit to make sure this is not possible.

Reply:
Looks like you removed the .json extension from the glob pattern so that should take care of it right? Maybe we could add a test that checks for 'no config file found' when there's a `lazy.config.json`

Reply:
Yeah, changing the glob pattern works fine. I was thinking if we need additional check somewhere deeper to prevent unwanted bundling of the unsupported files. But I guess we can keep it as it for now.